### PR TITLE
Update content and examples in docs

### PR DESCRIPTION
**Pull request by ryanchristo**
## Description

This pull request updates the content and examples in the docs, adding additional explanations, more information, cleaning up examples, and organizing the overall layout of the docs.

### CODE REVIEW

File: docs/_Docs_Transactions.md [hunk 1]
```diff
@@ -4,78 +4,86 @@ section: Docs
...
+## Multisignature Transactions
 
-These are for use with contract methods which require a transaction to be signed by multiple parties, specifically designed around [`simple-multisig`](https://github.com/christianlundkvist/simple-multisig). See [Multisig](https://docs.colony.io/tailor/docs-multisignature-transactions/) docs for more in depth explanation.
+The `multisig` transactions is used with contract methods which require a transaction to be signed by multiple parties. The `multisig` transaction was designed around [`simple-multisig`](https://github.com/christianlundkvist/simple-multisig). See [Multisignature Transactions](https://docs.colony.io/tailor/docs-multisignature-transactions/) in the [colonyJS docs](https://docs.colony.io/colonyjs/docs-overview/) for more information.
 
 ```js
+
 const client = await Tailor.load({
   methods: {
     myMethod: {
```
Comment (sprusr) [inline docs/_Docs_Transactions.md]: Ah my mistake, the `type` should actually be like this (with the options moved here):

```js
type: {
  name: 'multisig',
  options: {
    getRequiredSigners,
    multiSigFunctionName,
    getMultiSigNonce
  }
}
```

File: docs/_Docs_Wallets.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+Tailor supports a wide range of wallets with [Purser](https://github.com/JoinColony/purser), including both hardware and software wallets. See the [Purser Documentation](https://docs.colony.io/purser/docs-overview/) for more information on how wallets work, as well as a full list of wallets available.
+
+## How to use wallets
+
+With Purser, the configuration for the `Web3Wallet` is simple.
+
+### Web3 Wallet using MetaMask
+
+Import the `@colony/purser-metamsk` package, then execute the `open` method and pass the result to the `.load` method.
```
Comment (sprusr) [inline docs/_Docs_Wallets.md]: Little typo in the package name

File: docs/_Docs_Loaders.md [hunk 3]
```diff
@@ -73,16 +94,20 @@ const loader = new ContractHttpLoader({
   },
 });
 
-// The object may then be called by the adapter:
-const { abi, address, bytecode } = await loader.load({ contractAddress: '0xdeadbeef' });
+// Get the contract data using the loader
+const { abi, address, bytecode } = await loader.load({
```
[the remark this answers is not in the document]
Comment (ryanchristo) [reply docs/_Docs_Loaders.md]: 👍 Maybe an example with Tailor and a standalone example for each?

File: docs/_Docs_Loaders.md [hunk 4]
```diff
@@ -73,16 +94,20 @@ const loader = new ContractHttpLoader({
...
 
-Both the `EtherscanLoader` and `TrufflepigLoader` are modified versions of the more general `HttpLoader`. We hope to extend functionality to load from more data sources, such as:
+Both the `EtherscanLoader` and `TrufflepigLoader` are modified versions of the more general `HttpLoader`. We plan to extend this functionality to load contract data from more data sources, such as:
 
-- Databases (indexeddb on the browser, other dbs in node)
-- IPFS (which might be a very specific http loader)
-- Swarm
-- Browser file API (for testing in the browser)
+- IPFS
+- ENS packages
```
[the remark this answers is not in the document]
Comment (ryanchristo) [reply docs/_Docs_Loaders.md]: 👍 Whoops! That's what I meant to put in, not "ENS packages" 😅

File: docs/_Docs_Transactions.md [hunk 5]
```diff
@@ -4,78 +4,86 @@ section: Docs
 order: 5
 ---
 
-Interacting with the Ethereum blockchain in a way which will cause its state to change is done with a transaction. In Tailor we have various types of `Transaction` which handle everything for you surrounding sending a transaction.
+Interacting with the Ethereum blockchain in a way which will cause its state to change is done with a transaction. In Tailor we have various types of transactions, which handles everything for you when sending a transaction.
```
[the remark this answers is not in the document]
Comment (ryanchristo) [reply docs/_Docs_Transactions.md]: 👍

File: docs/_Docs_Transactions.md [hunk 6]
```diff
@@ -4,78 +4,86 @@ section: Docs
...
+  otherParam: 42,
+}).send();
+
 ```
 
-## MultiSig Transactions
+## Multisignature Transactions
 
-These are for use with contract methods which require a transaction to be signed by multiple parties, specifically designed around [`simple-multisig`](https://github.com/christianlundkvist/simple-multisig). See [Multisig](https://docs.colony.io/tailor/docs-multisignature-transactions/) docs for more in depth explanation.
+The `multisig` transactions is used with contract methods which require a transaction to be signed by multiple parties. The `multisig` transaction was designed around [`simple-multisig`](https://github.com/christianlundkvist/simple-multisig). See [Multisignature Transactions](https://docs.colony.io/tailor/docs-multisignature-transactions/) in the [colonyJS docs](https://docs.colony.io/colonyjs/docs-overview/) for more information.
```
[the remark this answers is not in the document]
Comment (ryanchristo) [reply docs/_Docs_Transactions.md]: 👍sssssome bad grammar here 😅

File: docs/_Docs_Transactions.md (hunk 1 above)
Comment (ryanchristo) [reply docs/_Docs_Transactions.md]: 👍

File: docs/_Docs_Wallets.md (hunk 2 above)
Comment (ryanchristo) [reply docs/_Docs_Wallets.md]: 👍

**Pull request merged**